examples/strings/strings.bal

```diff
@@ -11,47 +11,60 @@ public function main() {
     string s2 = statement.toLowerAscii();
     io:println("ToLower: ", s2);
 
-    // Creates a new `string`, which is a substring of the specified string. 
+    // Create a new `string`, which is a substring of the specified string.
     // You must provide the original `string`
     // and the starting and ending indexes of the substring.
     string s3 = statement.substring(0, 4);
-    io:println("SubString: " + s3);
+    io:println("SubString: ", s3);
 
-    // Retrieves the starting index of the first occurrence of the substring "on" within the `statement` string.
+    // Retrieve the starting index of the first occurrence of the substring "on" within the `statement` string.
     int? index = statement.indexOf("on");
     if (index is int) {
         io:println("IndexOf: ", index);
     }
-    
-    // Retrieves the length of the `string`.
+
+    // Retrieve the length of the `string`.
     int length = statement.length();
     io:println("Length: ", length);
 
-    // Removes any leading and trailing white spaces.
-    string s8 = statement.trim();
-    io:println("Trim: " + s8);
+    string hello = "Hello";
+    string ballerina = "Ballerina!";
+
+    // Concat multiple strings.
+    string s4 = hello.concat(" ", ballerina);
+    io:println("Concat: ", s4);
+
+    // Join strings with a separator.
+    string s5 = ",".'join(hello, ballerina);
+    io:println("Join: ", s5);
 
-    // Checks whether the given `string` ends with the suffix "Lion".
+    // Convert the `string` to a `byte` array.
+    byte[] bArray = hello.toBytes();
+
+    // Convert a `byte` array to a `string`.
+    string|error s6 = strings:fromBytes(bArray);
+    if (s6 is string) {
+        io:println("From bytes: ", s6);
+    }
+
+    // Remove any leading and trailing white spaces.
```

Comment:
```suggestion
    // Remove leading and trailing white spaces.
```

Reply:
Also `statement` does not have any leading and trailing white spaces right? Shall we declare another variable maybe with leading white spaces and use that instead?